### DISCUSSION TITLE
Registration failures from ueransim

### DISCUSSION THREAD

**Discussion by shashidhar-patil** [aligungr/UERANSIM#524] [Q&A]
Hello @aligungr,

**Problem:** Facing registration failures while testing 200 UE attaches with ueransim. 
GNB: sudo build/nr-gnb -c config/open5gs-gnb.yaml
UE: sudo build/nr-ue -c config/open5gs-ue.yaml -n **200**

In the following case we are getting only 187 requests on the AGW. The other 13 (200-187) are not being sent.

**Observations:**
UE side log for the failed ue. (IMSI - 901700000000051)
![1](https://user-images.githubusercontent.com/89975652/171414275-ecf3a1f6-e16d-4dba-bc59-8da919054570.jpg)

UE side log for successful ue. (IMSI - 901700000000001)
![2](https://user-images.githubusercontent.com/89975652/171414346-ef3cee93-8209-4933-a026-8b1c63c18a34.jpg)

Thank you in advance.

**Comment by GuilleCripto** [comment]
[2022-11-26 01:29:53.355] [nas] [error] Initial Registration failed [SEMANTICALLY_INCORRECT_MESSAGE]

**Comment by GuilleCripto** [comment]
excuse me, it gives me that error, in the ue
